src/gui/MainWindow.cc

```diff
@@ -403,7 +403,11 @@ MainWindow::MainWindow(const QStringList& filenames)
   connect(autoReloadTimer, SIGNAL(timeout()), this, SLOT(checkAutoReload()));
 
   this->exportformat_mapper = new QSignalMapper(this);
-  connect(this->exportformat_mapper, SIGNAL(mappedInt(int)), this, SLOT(actionExportFileFormat(int))) ;
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
+  connect(this->exportformat_mapper, &QSignalMapper::mappedInt, this, &MainWindow::actionExportFileFormat);
+#else
+  connect(this->exportformat_mapper, &QSignalMapper::mapped, this, &MainWindow::actionExportFileFormat);
```

Comment:
Trying to support ancient Ubuntu 20.04 too...
```suggestion
  connect(this->exportformat_mapper, static_cast<void (QSignalMapper::*)(int)>(&QSignalMapper::mapped), this, &MainWindow::actionExportFileFormat);
```
(see https://asmaloney.com/2015/05/code/qsignalmapper-example-revisited/)